beep/generate_protocol.py

```diff
@@ -1055,7 +1055,7 @@ def generate_maccor_waveform_file(df, file_prefix, file_directory, mwf_config=No
     return MWF_file_path
 
 
-def generate_protocol_files_from_csv(csv_filename, output_directory, **kwargs):
+def generate_protocol_files_from_csv(csv_filename, output_directory=None, **kwargs):
```

Comment:
Just curious about the reason for making this a default? is it for testing purposes?

Reply:
Ah, wait, I get it. This is fixing an inconsistency with the way the function is used later on

Reply:
Yeah, validation_records.json was always being written into the procedure_template folder despite having an output_directory argument.